packages/analytics-js/src/components/userSessionManager/UserSessionManager.ts

```diff
@@ -286,7 +354,7 @@ class UserSessionManager implements IUserSessionManager {
       }
 
       if (autoTrack) {
-        state.session.sessionInfo.value = { ...defaultSessionInfo };
+        state.session.sessionInfo.value = {};
```

Comment:
this is the case that you asked about desired logic a couple of days ago?

Reply:
Not this one. That is for endSession().